packages/website/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { Box } from '@coderscamp/ui/components/Box';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { Logo } from '@coderscamp/ui/components/Logo';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import {
+  SolidFacebookIcon,
+  SolidGitHubIcon,
+  SolidGlobalIcon,
+  SolidInstagramIcon,
+  SolidLinkedinIcon,
+} from '@coderscamp/ui/icons';
+import { colors } from '@coderscamp/ui/theme/overwrites/foundations/colors';
+
+export const socialMediaIcons = [
+  { name: 'FacebookIcon', component: <SolidFacebookIcon /> },
+  { name: 'InstagramIcon', component: <SolidInstagramIcon /> },
+  { name: 'LinkedinIcon', component: <SolidLinkedinIcon /> },
+  { name: 'GitHubIcon', component: <SolidGitHubIcon /> },
+  { name: 'GlobalIcon', component: <SolidGlobalIcon /> },
+];
+
+export const Footer = () => {
+  const footerStyles = {
+    upperFooter: {
+      direction: useBreakpointValue({ base: 'column', lg: 'row' } as const),
+    } as const,
+    lowerFooter: {
+      direction: useBreakpointValue({ base: 'column', md: 'row' } as const),
+      order: useBreakpointValue({ base: -1, md: 1 } as const),
```

Comment:
💭 You don't have to keep those styles as objects. You can write them as:
```
const footerDirection = useBreakpointValue({ base: 'column', lg: 'row' } as const);
const lowerFooterOrder = useBreakpointValue({ base: 'column', lg: 'row' } as const);
```

Reply:
I think that object is easier for me and in future using.